BrentOzarULTD/SQL-Server-First-Responder-Kit
# Add sp_Blitz's parameter @OutputType to sp_BlitzIndex and sp_BlitzFirst

Fixes #  #1679

Changes proposed in this pull request:
 - some tab to space adjustments
 - some reformatting
 - add a parameter @Output parameter or the handling of a "NONE" value for this parameter

How to test this code:
 - same as usual should give the same result as usual
- set @OutputType to NONE alone will make it fail
 - set @OutputType to NONE with @OutputTableName to something will store the results to the output table 

Has been tested on (remove any that don't apply):
 - Case-sensitive SQL Server instance
 - SQL Server 2014
